src/index.html

```diff
@@ -5,9 +5,30 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <h1></h1>
+    <div class="card" data-qa="card">
+      <div class="card__image"></div>
+      <div class="card__content">
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="card__stars">
+            <span class="card__star"></span>
+            <span class="card__star"></span>
+            <span class="card__star"></span>
+            <span class="card__star"></span>
+            <span class="card__star"></span>
+          </div>
+          <p class="card__reviews-text">Reviews: 5</p>
+        </div>
+        <p class="card__price">Price: <span class="card__price-value">$2,199</span></p>
+        <a href="#"
+          class="card__link" data-qa="hover">Buy</a>
```

Comment:
apply code styling recommendations from a task checkilist
```suggestion
        <a 
          href="#"
          class="card__link" 
          data-qa="hover"
        >
          Buy
        </a>
```